budget/app.py

```diff
@@ -8,38 +10,64 @@
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
-# TODO configure this URL?
-SYNAPSE_TEAM_MEMBER_URL = 'https://repo-prod.prod.sagebase.org/repo/v1/teamMembers'
-
 # TODO pull this config from S3?
-USER_BUDGET_BY_TEAM = {
-  '12345': {
-    'amount': 2000,
-    'period': 'annual'
+# and validate it with jsonschema in a future PR
+# this configuration will differ between accounts
+
+TEAM_BUDGET_RULES = {
+  'teams': {
+    '3412821': {
+      #'amount': '2000',
+      'amount': '10',
+      'period': 'ANNUALLY',
+      'community_manager_emails': ['tess.thyer@sagebase.org']
```

Comment:
I suggest we use @synapse.org email addresses whenever possible.... does this design actually require publicly resolvable addresses?

Reply:
We can't send random mails to synapse.org addresses -- they can only be sent through the Synapse system. That's why SNS is used to send the message to the user. At a later stage in development when this configuration gets moved to multiple files, the community manager emails will probably be changed to something like sc-support@sagebase.org.